### ISSUE TITLE
Request: deck cost

### ISSUE THREAD

**Issue by riser88** [Epix37/Hearthstone-Deck-Tracker#905]
Hello,

There is lot of new players that don't have enought dust to collect their deck. It would be fantastic to create another deck category for such decks so at the end there would be:
* arena deck
* constructed deck
* planned deck

In "planned deck" tab next to deck name should be displayed deck cost in dust with additional information about required expansion (GvG, Nax , BRM , etc ...). It would also to see number of cards that we arleady have. On top of that there should be sorting option (by cost & by number of missing cards)

For example:
* Low Budget Warrior Grim Patron OTK (One Turn Kill) BrM Deck - 580 dust - NAX - 24/30

Best Regards :)

**Comment by skaforey** [comment]
I don't know the code that well but this seems like it would require extensive changes.

Either HDT would have to know all of the cards you own, or you would manually have to "mark" them on a per deck basis.  I know there are websites around there that do keep track of your cards and provide similar functionality to what you're asking for, so it might easier to use one of those.

But who knows, epix is amazing, so maybe it's easier than I envisioned :)

**Comment by culinko** [comment]
I don't think there is a need for a new tab. All of the "planned" decks are constructed decks anyway, so maybe just group them on the bottom of the list with some header or something like that. They could be tagged with new "planned" quick tag, too.

**Comment by Epix37** [comment]
I agree a new tab isn't really necessary here. Tags should do it perfectly fine (you can create new tags via `DECK > EDIT TAGS`). 

Displaying the missing cards after exporting should work with the next update properly agian, too.

As for dust cost, that's something that would probably look decent in the (new) deck stats somewhere. Adding the new sorting filters shouldn't be much of a problem either.

**Comment by Cu8eR** [comment]
@Epix37 what about to show only total dust value of all cards in your deck?

**Comment by Epix37** [comment]
Isn't that what we are talking about? That's what I was talking about anyway.

**Comment by Cu8eR** [comment]
They wanna show how many dust they need for crafting missing cards - this is a little bit difficult to implement

I just wanna show total value all deck

**Comment by Epix37** [comment]
Not how I understood it, but that's what I plan on doing, yea. 

The (now fixed) exporting shows the missing cards and their cost already anyway.

**Comment by MeSko** [comment]
Can you add dust cost next to missing card name.
Thank you very much.
